server/service/src/sync/init_programs_data.rs

```diff
@@ -1131,7 +1143,7 @@ pub fn init_program_data(
             name: "List of Appointments".to_string(),
             r#type: repository::ReportType::OmSupply,
             template: ENCOUNTERS_REPORT.to_string(),
-            context: ReportContext::Dispensary,
+            context: ReportContext::Patient,
```

Comment:
why the change? 
There are now no results in the dispensary report list

Reply:
Not sure how these changes magically appeared 👀 

[641ed57](https://github.com/openmsupply/open-msupply/pull/1262/commits/641ed579eb72df6500624a298763daeadffea166)